frontend/src/context/mapStateSlice/index.ts

```diff
@@ -44,10 +44,28 @@ function keepLayer(layer: LayerType, payload: LayerType) {
   // Simple function to control which layers can overlap.
   return (
     payload.id !== layer.id &&
-    (payload.type !== layer.type || payload.type === 'boundary')
+    (payload.type !== layer.type ||
+      TYPES_ALLOWED_TO_OVERLAP.includes(payload.type))
   );
 }
 
+// Order layers to keep boundaries and point_data on top. point_data first.
+function layerOrdering(a: LayerType, b: LayerType) {
```

Comment:
@wadhwamatic this allows us to order layers.
Right now it puts point_data, then admin boundaries, and then it doesn't care. Can you think of other layers we would want to prioritize?

Reply:
After point_data, I think the ideal state would be show whichever layer was most recently selected next.